odoo/addons/test_http/tests/test_registry.py

```diff
@@ -86,7 +86,7 @@ def url_open(self, path, *, allow_redirects=False):
         if not path.startswith('/'):
             raise ValueError("can only request a relative url")
         url = urljoin(f"http://{HOST}:{odoo.tools.config['http_port']}", path)
-        return self.opener.get(url, allow_redirects=allow_redirects)
```

Comment:
Revert the change, you are introducing an infinite recursion.
This module is about testing the low-level database operations (create/duplicate/delete), I would advise against changing it too much as it is kinda sensitive.

Reply:
Good catch 👍